common/io/SelectServerTest.cpp

```diff
@@ -582,7 +582,7 @@ void SelectServerTest::testTimeout() {
       10,
       ola::NewSingleCallback(this, &SelectServerTest::ReentrantTimeout, m_ss));
   m_ss->RegisterSingleTimeout(
-      20,
+      40,
```

Comment:
Hmm, ok. Can you add a comment that references this PR so we know why it's so high.